draft-ietf-quic-http.md

```diff
@@ -520,9 +617,9 @@ permitted (e.g., idempotent actions like GET, PUT, or DELETE).
 ### Malformed Requests and Responses {#malformed}
 
 A malformed request or response is one that is an otherwise valid sequence of
-frames but is invalid due to the presence of extraneous frames, prohibited
-header fields, the absence of mandatory header fields, or the inclusion of
-uppercase header field names.
+frames but is invalid due to the presence of prohibited header fields, the
+absence of mandatory header fields, an invalid sequence of HTTP messages, or the
+inclusion of uppercase header field names.
```

Comment:
You don't mention pseudo-header fields in this paragraph but I think it would help to do so (because earlier it says `Pseudo-header fields are not HTTP header fields.`).